lang/繁體中文.json

```
@@ -156,6 +162,6 @@
             "how_it_works_text": "每隔一段時間，應用程式就會向 Twitch 請求當前觀看頻道的原始串流數據的 URL。然後提取此串流的元數據——這足以推進掉寶進度。請注意，這完全繞過了下載實際的串流視頻和音頻的需求。為了使頻道的狀態（上線或離線）的即時更新，應用程式建立了一個 WebSocket 連接，該連接會接收有關串流上下線的事件或當前觀眾數量的更新。",
             "getting_started": "事前準備",
             "getting_started_text": "1. 透過應用程式登入。\n2. 確保您的 Twitch 帳號已經和您所有感興趣的遊戲帳號已經連接。\n 3. 如果您想要獲取所有感興趣的掉寶，請不要勾選「僅參與優先掉寶遊戲」，然後按「重新載入」。\n4. 如果您想優先獲取特定遊戲，請使用「優先掉寶遊戲」列表設置您想獲取的遊戲的優先順序。應用程式將嘗試先獲取列表頂端的遊戲，然後依序獲取之後的遊戲。\n5. 勾選「僅參與優先掉寶遊戲」以避免獲取不在優先列表中的遊戲。當然，你也可以選擇不勾選——這取決於您。\n6. 使用「排除掉寶遊戲」列表設置永不獲取的遊戲。\n7. 更改任一列表的內容，或更改「僅參與優先掉寶遊戲」勾選狀態，需要手動按「重新載入」才能使更改生效。"
-		}
+        }
```

Actually, it's best to keep this as tabs, like it was before, to avoid the current diff, which basically replaces tabs with spaces. This puts it back as tabs. I'll normalize everything to spaces after this is merged.

```suggestion
		}
```

---

I thought this was Github automatically use Tab space when I hit Enter, I'll change it.